### DISCUSSION TITLE
How to watch defaultValue defined in Controller (not useForm)

### DISCUSSION THREAD

**Discussion by slillo** [react-hook-form/react-hook-form#5713] [Q&A]
I'm trying to "watch" value of another field, but default value is ignored because I provide default value via Controller prop (not useForm).

Here is a [demo](https://codesandbox.io/s/fervent-rosalind-u0jyg?file=/src/App.js). 

Here I'm trying to disable a field if another field is empty. 

As you can see the second field is disabled also if first field is not empty. The second field remains disabled until I manually edit first field (without emptying it) or if defaultValue is set via useForm.

Maybe this is the expected behaviour but I really need to watch also for initial value (defaultValue) when it is set via Controller prop. How?

Thanks in advance.

**Comment by bluebill1049** [comment]
for inline defaultValue, it's impossible for the first render to return the correct defaultValue, you will have to provide a defaultValue to the watch, because watch will run before the render.
